src/SmartComponents/PatchManager/PatchManagerCard.js

```diff
@@ -1,51 +1,106 @@
 
 /**
  * Operating systems card for showing the ratio of operating systems used.
  */
-const PatchManagerCard = () => {
+const PatchManagerCard = ({ systems, systemsStatus, fetchSystems, fetchSecurity, securityStatus,
+    security, bugs, fetchBugs, bugsStatus, enhancements, fetchEnhancements, enhancementsStatus, intl }) => {
+
+    React.useEffect(() => {
+        fetchSystems();
+        fetchSecurity();
+        fetchBugs();
+        fetchEnhancements();
+    }, [fetchSystems, fetchSecurity, fetchBugs, fetchEnhancements]);
+
+    const isLoaded = [systemsStatus, securityStatus, bugsStatus, enhancementsStatus].every(item => item === 'fulfilled');
+
     const pieChartData = [
-        { x: 'security advisories', y: 254, fill: '#004b95' },
-        { x: 'bug fixes', y: 400, fill: '#06c' },
-        { x: 'enhancements', y: 100, fill: '#519de9' }
+        { x: intl.formatMessage(messages.securityAdvisories, { count: security }), y: security, fill: '#004b95' },
+        { x: intl.formatMessage(messages.bugfixAdvisories, { count: bugs }), y: bugs, fill: '#06c' },
+        { x: intl.formatMessage(messages.enhancementAdvisories, { count: enhancements }), y: enhancements, fill: '#519de9' }
     ];
     const pieChartLegendData = pieChartData.map(item => ({ name: `${item.y} ${item.x}`, symbol: { fill: `${item.fill}`, type: 'circle' } }));
     const colorScale = ['#004b95', '#06c', '#519de9'];
     const pieChartPadding = { bottom: 0, left: 0, right: 220, top: 0 };
     return <TemplateCard appName='PatchManager'>
         <TemplateCardHeader subtitle='Patch manager'/>
         <TemplateCardBody>
-            <Button
-                component="a"
-                href=""
-                variant="link"
-                isInline
-                style={ { textAlign: 'left' } }
-            >
-                0 systems affected
-            </Button>
-            <PieChart
-                className="ins-c-pie-chart"
-                containerWidth={ 275 }
-                containerHeight={ 90 }
-                ariaDesc="Operating systems used"
-                ariaTitle="Pie chart operating systems"
-                constrainToVisibleArea={ true }
-                data={ pieChartData }
-                height={ 150 }
-                labels={ ({ datum }) => `${datum.x}: ${datum.y}` }
-                legendData={ pieChartLegendData }
-                legendOrientation="vertical"
-                legendPosition="right"
-                padding={ pieChartPadding }
-                width={ 290 }
-                colorScale={ colorScale }
-            />
+            {!isLoaded ? <Loading/> :
+                <React.Fragment>
+                    <Button
+                        component="a"
+                        href={ `${UI_BASE}/${PATCHMAN_ID}/systems` }
+                        variant="link"
+                        isInline
+                        style={ { textAlign: 'left', zIndex: 3 } }
```

Comment:
why does this need a z-index of 3?

Reply:
re this, requesting that we apply the styles to a class defined in a colocated .scss

Reply:
I forgot to mention this one... For some reason, the link is not hoverable without z-index override but honestly I didn't dig very deep to find out what causes it. I blame PF Pie Chart which has this in it 
![image](https://user-images.githubusercontent.com/6286045/77356504-b892fb00-6d46-11ea-9c84-8b4e256b9c19.png)

Reply:
I actually just updated the pie chart in this PR: https://github.com/RedHatInsights/insights-dashboard/pull/72 and now the link will work without the z-index

<img width="270" alt="Screen Shot 2020-03-23 at 3 55 07 PM" src="https://user-images.githubusercontent.com/20118816/77357517-b157f700-6d1e-11ea-80a1-939965766342.png">

Reply:
Removed z-index and moved the style to the separate file.